nilearn/glm/first_level/first_level.py

```diff
@@ -220,6 +220,33 @@ def run_glm(Y, X, noise_model='ar1', bins=100,
         results = {0.0: ols_result}
 
     return labels, results
+  
+def check_trial_type(df):
+  
+    """ Raises warning if the dataframe corresponding to 
+      to events.tsv doesn't contain column named "trial_type".
+      
+    parameters
+    ----------
+    df : pandas dataframe corresponding to a events.tsv file
+
+    """
+    if len(df.columns) != 2 and 'trial_type' not in df.columns:
```

Comment:
```suggestion
    if len(df.columns) > 2 and 'trial_type' not in df.columns:
```

If the value is inferior to 2, then this is not even a BIDS valid dataset because the events file must at least have `onset` and `duration`.

Better if we stick to cases where the value is more than 2 because otherwise we start doing some sort of indirect BIDS validation: let's make our life easier and assume users have a valid BIDS dataset as input.